playbooks/roles/run-zuul/templates/main.yaml.j2

```diff
@@ -1,5 +1,5 @@
 - tenant:
-    name: Tungsten Fabric
+    name: CodiLime
```

Comment:
Tego nie wolno bo gerrit przestanie działać. Poza tym jestem prawie pewien ze to zmieniłem na "acid".